public/index.html

```diff
@@ -1,51 +1,57 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Gleam Playground</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1.00, maximum-scale=1.00, minimum-scale=1.00, user-scalable=no"/>
-    </head>
-    <body>
-        <div class="spa">
-            <div class="header">
-                <h2>Gleam Playground</h2>
-                <a href="https://github.com/JohnDoneth/gleam-playground">
-                    <img src="github.png" class="gh-link"/>
-                </a>
+
```

Comment:
As an aside: These formatting changes remind me that we should add this file to the prettier search path for CI.

Reply:
I don't know how to do that, but I formatted the HTML file with prettier, so it should be fine now.